Upstream: no
Reason: Fix underlinking due to bctoolbox symbol usage.

diff -urN sng-tc-linux-1.3.9.i686.orig/client/sngtc_client/Makefile sng-tc-linux-1.3.9.i686/client/sngtc_client/Makefile
--- sng-tc-linux-1.3.9.i686.orig/client/sngtc_client/Makefile	2020-04-30 13:17:32.005717991 -0600
+++ sng-tc-linux-1.3.9.i686/client/sngtc_client/Makefile	2020-04-30 13:17:53.909139976 -0600
@@ -7,9 +7,9 @@
 NO_NODE = 0
 
 ifeq ($(NO_NODE),0)          
-LIBS = -lsngtc_node -lrt -lortp -lpthread
+LIBS = -lsngtc_node -lrt -lortp -lpthread -lbctoolbox
 else
-LIBS = -lsng-tc -lrt -lortp -lpthread
+LIBS = -lsng-tc -lrt -lortp -lpthread -lbctoolbox
 CFLAGS += -DSNGTC_LIB_BIND
 endif
 
